src/reference/environments/rose.yml

```diff
@@ -0,0 +1,10 @@
+name: cylc-8-with-rose
+# Cylc 8, the UI server, Cylc Rose and Metomi Rose
+channels:
+  - conda-forge
+dependencies:
+  - cylc
+  - cylc-rose
+  - cylc-ui
+  - cylc-uiserver
+  - metomi-rose
```

Comment:
This is using the old meta package (which includes all of the dependencies specified below at earlier versions) and installing the cylc-ui which is not needed.